handler/OnAgentAssignedHandler.ts

```diff
@@ -52,9 +54,18 @@ export class OnAgentAssignedHandler {
 
         await updateRoomCustomFields(rid, { welcomeEventSent: true }, this.read, this.modify);
 
+        const assoc = new RocketChatAssociationRecord(RocketChatAssociationModel.MISC, `SFLAIA-${rid}`);
```

Comment:
We should create a common Utility function that takes roomId as an argument and returns the `RocketChatAssociationRecord` object. And reuse that across the app.

And we should rename `SFLAIA-` to something more meaningful.